MHVLib/MHVUser.h

```diff
@@ -24,80 +24,71 @@
 
 //
 // The user of your HealthVault application
-// 
+//
 @interface MHVUser : XSerializableType
-{
-@private
-    NSString * m_name;
-    MHVRecordCollection *m_records;
-    NSInteger m_currentIndex;
-    NSString* m_environment;
-    NSString* m_instanceID;
-}
 
 //
 // The user's display name in HealthVault
 //
-@property (readwrite, nonatomic, strong) NSString* name;
+@property (readwrite, nonatomic, strong) NSString *name;
 //
 // Records this user is authorized to access
 //
-@property (readwrite, nonatomic, strong) MHVRecordCollection* records;
+@property (readwrite, nonatomic, strong) MHVRecordCollection *records;
 @property (readonly, nonatomic) BOOL hasRecords; // true if user has authorized records
 //
 // The records the application is currently working with
 // To change the current record, set the currentRecordIndex property
 //
 @property (readwrite, nonatomic) NSInteger currentRecordIndex;
-@property (readonly, nonatomic, strong) MHVRecord* currentRecord;
+@property (readonly, nonatomic, strong) MHVRecord *currentRecord;
 
 //
 // Which service environment this user set up their app to work with
 //
-@property (readwrite, nonatomic, strong) NSString* environment;
+@property (readwrite, nonatomic, strong) NSString *environment;
 
-@property (readwrite, nonatomic, strong) NSString* instanceID;
+@property (readwrite, nonatomic, strong) NSString *instanceID;
 
 @property (readonly, nonatomic) BOOL hasEnvironment;
 @property (readonly, nonatomic) BOOL hasInstanceID;
 
-//-------------------------
+// -------------------------
 //
 // Methods
 //
-//-------------------------
+// -------------------------
 
--(id) initFromLegacyRecords:(NSArray *) recordArray;  // Infrastructure - will eventually go away
```

Comment:
Let's leave this comment in here until we can determine if this has been deprecated.